### PR TITLE
iObeya

### PR DESCRIPTION
**Pull request by smiquerolle**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

File: certified-connectors/iObeya/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,598 @@
...
+          }
+        },
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/qcd/indicators-values": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default"
```
Comment (v-EgorKozhadei) [inline certified-connectors/iObeya/apiDefinition.swagger.json:355]: @smiquerolle please provide schemas for all action operations. More information about response schemas [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses).

File: certified-connectors/iObeya/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,598 @@
...
+            "description": "Indicator Name",
+            "x-ms-summary": "Indicator Name"
+          },
+          {
+            "name": "wedgeValue",
+            "in": "query",
+            "required": true,
+            "type": "number",
+            "description": "Indicator Value",
+            "x-ms-summary": "Indicator Value"
```
Comment (v-EgorKozhadei) [inline certified-connectors/iObeya/apiDefinition.swagger.json:434]: @smiquerolle summaries and descriptions should not look like a copy and paste. Please refer to English strings [guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

File: certified-connectors/iObeya/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,598 @@
...
+        "summary": "List Cards Activity",
+        "description": "List Cards Activity between two dates",
+        "operationId": "ListCardsActivity",
+        "parameters": [
+          {
+            "name": "boardId",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Board where retrieve activity card",
```
Comment (v-EgorKozhadei) [inline certified-connectors/iObeya/apiDefinition.swagger.json:536]: @smiquerolle please make sure everything is grammatically correct and is free of typographical errors.

File: certified-connectors/iObeya/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+          "tooltip": "The API Client Secret created in iObeya Administration > Settings > API"
+        }
+      }
+    },
+    "iconBrandColor": "#f8faff",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "setproperty",
+        "title": "Create a nextLink",
```
Comment (v-EgorKozhadei) [inline certified-connectors/iObeya/apiProperties.json:73]: @smiquerolle strings with camel case are considered non-translatable. If you want to localize the string, the camel case should be fixed here. More information about English strings can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

File: certified-connectors/iObeya/README.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    - Standard
+    - Activity
+    - Feature
+    - Story
+    - ProblemSolving
+    - QCD Action
+* `QCD Indicators Bulk Update`: Bulk update your QCD indicators from various data sources (Excel, SAP, etc..)
+* `List activity card`: Retrieve all activity cards by date range from a planning board
+
+## Contact us
```
[the remark this answers is not in the document]
Comment (smiquerolle) [reply certified-connectors/iObeya/README.md:32]: @v-EgorKozhadei  There is no "Contact" section in the template. Where we can put the contact email ?

**Comment by smiquerolle** [comment]
Hi @v-EgorKozhadei , I made some changes regarding your feedback. It's hard to find good "English strings", let me know if they need more attention.

File: certified-connectors/iObeya/README.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
+# iObeya Connector
```
[the remark this answers is not in the document]
Comment (smiquerolle) [reply certified-connectors/iObeya/README.md]: Done

File: certified-connectors/iObeya/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (smiquerolle) [reply certified-connectors/iObeya/README.md]: Thanks

File: certified-connectors/iObeya/apiDefinition.swagger.json [hunk 7]
```diff
@@ -0,0 +1,598 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "iObeya",
+    "description": "Uses the iObeya API to integrate your data.",
+    "version": "1.0.0",
```
[the remark this answers is not in the document]
Comment (smiquerolle) [reply certified-connectors/iObeya/apiDefinition.swagger.json]: Updated

File: certified-connectors/iObeya/apiDefinition.swagger.json [hunk 8]
```diff
@@ -0,0 +1,598 @@
...
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/rooms": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "List of rooms",
```
[the remark this answers is not in the document]
Comment (smiquerolle) [reply certified-connectors/iObeya/apiDefinition.swagger.json]: Done

File: certified-connectors/iObeya/apiDefinition.swagger.json [hunk 9]
```diff
@@ -0,0 +1,598 @@
...
+              "type": "object",
+              "properties": {
+                "self": {
+                  "type": "string",
+                  "description": "self",
+                  "title": "Self Collection URL"
+                },
+                "kind": {
+                  "type": "string",
+                  "description": "",
```
[the remark this answers is not in the document]
Comment (smiquerolle) [reply certified-connectors/iObeya/apiDefinition.swagger.json]: I filled all empty properties.

**Review by v-EgorKozhadei — APPROVED** [review]

**Comment by v-EgorKozhadei** [comment]
Hello @smiquerolle, 

I hope you are doing well. 

Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) using your latest commit id, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. 

After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6. 

Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.  

We expect all tests to be completed within 1-2 weeks. 

If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions. 

Please let me know if you have any questions. 

Thank you very much for working with us.

**Pull request merged**